src/ledger.js

```diff
@@ -0,0 +1,131 @@
+// LedgerSystem class
+const BigNumber = require('bignumber.js');
+const Data = require("./database");
+const FIFOQueue = require("./queue");
+const sleep = require('./utils').sleep;
+const groupBy = require('./utils').groupBy;
+const OperationTypes = {
+    TRANSFER: "TRANSFER"
+}
+const OperationStatus = {
+    INIT: "INIT",
+    PROCESSING: "PROCESSING",
+    APPLIED: "APPLIED",
+    REJECTED: "REJECTED"
+}
+const Entities = {
+    OPERATIONS: "operations",
+    ACCOUNTS: "accounts",
+    ENTRIES: "entries"
+}
+class LedgerSystem {
+    // singleton class
```

Comment:
How is it singleton?

Reply:
There will be only one ledgerSystem instance in the server. as there needs to be only one FIFOQueue